javascript-modules-engine/tests/package.json

```diff
@@ -20,6 +20,7 @@
         "@jahia/cypress": "^4.2.0",
         "@jahia/jahia-reporter": "^1.0.30",
         "@jahia/jcontent-cypress": "^3.0.0-tests.8",
+        "@types/mocha": "^10.0.10",
```

Comment:
where is that new dependency used? do we actually need it? 🤔

Reply:
It provides autocompletion and type hints in .cy.ts files, it does not contains any code

<img width="699" alt="image" src="https://github.com/user-attachments/assets/71f4dd4f-0955-46dc-9998-19b9b45afa5b" />

If you have it working already without `@types/mocha` I'm curious to know your setup

Reply:
In IntelliJ, it seems to work fine 🤷‍♂️ :
<img width="947" alt="image" src="https://github.com/user-attachments/assets/e411a123-e25a-4b43-b5cc-cded1e52d235" />
Looks like @types/mocha is pulled transitively by Cypress. Anyway, definitely cleaner to have it correctly declared as dev dependency!